README.md

```diff
@@ -62,12 +62,13 @@ sudo apt-get install make cmake g++ libboost-dev libboost-system-dev \
   libbz2-dev libpq-dev libproj-dev lua5.2 liblua5.2-dev
 ```
 
-On a Fedora system, use
+On a Fedora 24/25 system, use
 
 ```sh
-sudo yum install cmake gcc-c++ boost-devel expat-devel zlib-devel bzip2-devel \
-  postgresql-devel proj-devel proj-epsg lua-devel
+sudo dnf install cmake make gcc-c++ boost-devel expat-devel zlib-devel bzip2-devel \
+  postgresql-devel proj-devel proj-epsg lua-devel geos-devel
```

Comment:
Is dnf also used in the current stable CentOS release? If yes, you should be changing the EPFL installation instructions below, if not then the CentOS section below needs explicit installation instructions using yum.

Reply:
Checked centos:7, dnf is *not* installed by default.